packages/type-system/rust/build-wasm.py

```diff
@@ -0,0 +1,59 @@
+import subprocess
+import json
+from pathlib import Path
+from typing import Dict
+import logging
+
+
+def run_wasm_pack(target, out_dir: Path):
+    logging.debug("Running wasm-pack, targeting %s, for %s", target, out_dir)
+    subprocess.run(["wasm-pack", "build", "--target", target, "--out-dir", str(out_dir), "--out-name", "index", "--scope", "blockprotocol"], check=True)
+
+
+def patch_package_json(file: Path, unique_patch: Dict):
```

Comment:
There aren't great configuration options for the generated `package.json`s and if we're publishing to NPM I assumed we'd want to patch in some content. I wrote this script to setup the infra to do so but I don't really know what we want to patch in.

Reply:
This has been made redundant